src/main/java/ppp/discord/bot/commands/PointsCommand.java

```diff
@@ -0,0 +1,82 @@
+package ppp.discord.bot.commands;
+
+import discord4j.core.event.domain.message.MessageCreateEvent;
+import discord4j.core.object.entity.Message;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import ppp.discord.bot.model.User;
+import ppp.discord.bot.utils.LanguageUtil;
+import reactor.core.publisher.Mono;
+
+@Component("pointsCommand")
+public class PointsCommand implements Command<MessageCreateEvent> {
```

Comment:
Try to stick to 1 feature, this commit touches both on the points  and language feature  =)